commands/anonSuggest.js

```diff
@@ -0,0 +1,39 @@
+const {GuildChannel} = require('discord.js');
+
+module.exports = {
+	name: 'anonsuggest',
+	description: 'Posts an anonymous Suggestion',
+	usage: 'anonsuggest',
+	helpMsg: true,
+	execute(message, args) {
+		const {modBot} = message.client.config.channelIDs;
+		const suggestion = args.join(' ');
+
```

Comment:
I will flame you for this needless empty line

Reply:
The others came along later.